File: Ser2025/Company/Company.Web/Modules/ServerTypes/Company/AddressForm.ts
import { PrefixedContext, IntegerEditor, StringEditor, initFormType } from '@serenity-is/corelib';

export interface AddressForm {
    AddressLine1: StringEditor;
    AddressLine2: StringEditor;
    City: StringEditor;
    CountryCode: StringEditor;
    StateProvinceId: IntegerEditor;
    PostalCode: StringEditor;
    IsActive: IntegerEditor;
    TenantId: IntegerEditor;
}

export class AddressForm extends PrefixedContext {
    static readonly formKey = 'Company.Address';
    private static init: boolean;
    
    constructor(prefix: string) {
        super(prefix);
        if (!AddressForm.init)  {
            AddressForm.init = true;
            
            var w0 = IntegerEditor;
            var w1 = StringEditor;

            initFormType(AddressForm, [
            'AddressLine1', w1,
            'AddressLine2', w1,
            'City', w1,
            'CountryCode', w1,
            'StateProvinceId', w0,
            'PostalCode', w1,
            'IsActive', w0,
            'TenantId', w0,
            ]);
        }
    }
}

File: Ser2023/temporaly/Address/AddressForm.cs
using Serenity.ComponentModel;
using Serenity.Web;
using System;

namespace Address.Address.Forms
{
    [FormScript("Address.Address")]
    [BasedOnRow(typeof(AddressRow), CheckNames = true)]
    public class AddressForm
    {
        public string AddressLine1 { get; set; }
        public string AddressLine2 { get; set; }
        public string City { get; set; }
        public int StateProvinceId { get; set; }
        public string PostalCode { get; set; }
        public Guid Rowguid { get; set; }
        public short IsActive { get; set; }
        public int TenantId { get; set; }
    }
}